assets/xml/textures/nes_font_static.xml

```diff
@@ -14,130 +14,131 @@
         <Texture Name="gMsgCharPlusSignTex" OutName="msg_char_plus_sign" Format="i4" Width="16" Height="16"/>
         <Texture Name="gMsgCharCommaTex" OutName="msg_char_comma" Format="i4" Width="16" Height="16"/>
         <Texture Name="gMsgCharHyphenMinusTex" OutName="msg_char_hyphen_minus" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharPeriodTex" OutName="msg_char_period" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgCharSlashTex" OutName="msg_char_slash" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar0Tex" OutName="msg_char_0" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar1Tex" OutName="msg_char_1" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar2Tex" OutName="msg_char_2" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar3Tex" OutName="msg_char_3" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar4Tex" OutName="msg_char_4" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar5Tex" OutName="msg_char_5" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar6Tex" OutName="msg_char_6" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar7Tex" OutName="msg_char_7" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar8Tex" OutName="msg_char_8" Format="i4" Width="16" Height="16"/>
-        <Texture Name="gMsgChar9Tex" OutName="msg_char_9" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharFullStopTex" OutName="msg_char_full_stop" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharSolidusTex" OutName="msg_char_solidus" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitZeroTex" OutName="msg_char_digit_zero" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitOneTex" OutName="msg_char_digit_one" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitTwoTex" OutName="msg_char_digit_two" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitThreeTex" OutName="msg_char_digit_three" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitFourTex" OutName="msg_char_digit_four" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitFiveTex" OutName="msg_char_digit_five" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitSixTex" OutName="msg_char_digit_six" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitSevenTex" OutName="msg_char_digit_seven" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitEightTex" OutName="msg_char_digit_eight" Format="i4" Width="16" Height="16"/>
+        <Texture Name="gMsgCharDigitNineTex" OutName="msg_char_digit_nine" Format="i4" Width="16" Height="16"/>
```

Comment:
Can I ask that some synchronisation be put in place to match the names of the textures, with the names used in code, and the names/ordering of the character width values.

It will make modding easier, but also help make it easier to read the code, and get what matches with what.

Reply:
Agreed, whichever we use should be consistent between Name and OutName. I assume Unicode has a rule about using only capital letters and spaces in its names.

Reply:
>A Unicode character is assigned a unique Name (na).[1] The name is composed of uppercase letters A–Z, digits 0–9, - (hyphen-minus) and <space>. Some sequences are excluded: names beginning with a space or hyphen, names ending with a space or hyphen, repeated spaces or hyphens, and space after hyphen are not allowed. The name is guaranteed to be unique within Unicode, and can be used to identify a code point and its character. Ideographic characters, of which there are tens of thousands, are named in the pattern "cjk unified ideograph-hhhh". For example, ``U+4E00 一 CJK UNIFIED IDEOGRAPH-4E00``. Formatting characters are named too: ``U+00A0   NO-BREAK SPACE``.
>

Reply:
I'm happy to use digits 0-9. But if we use codepoints in the names as well, it won't matter nearly so much.

Reply:
Yeah, I came with that idea later. I think I prefer the hex preffix.